### PR TITLE
Add configuration option to whitelist all exceptions.

### PR DESCRIPTION
**Pull request by Petesta**

### CODE REVIEW

**Comment by Petesta** [comment]
This PR addresses this [issue](https://github.com/cerebris/jsonapi-resources/issues/887).

**Comment by Petesta** [comment]
@lgebhardt

File: lib/jsonapi/configuration.rb [hunk 1]
```diff
@@ -81,6 +82,11 @@ def initialize
       # the `Pundit::NotAuthorizedError` to the `exception_class_whitelist`.
       self.exception_class_whitelist = []
 
+
```
Comment (lgebhardt) [inline lib/jsonapi/configuration.rb]: A minor nit, but could you remove this extra line?

Comment (Petesta) [reply lib/jsonapi/configuration.rb]: Ahh, yes!

**Comment by lgebhardt** [comment]
@Petesta Thanks!

**Comment by Petesta** [comment]
Is there a plan to publish version `0.9.0.pre` to rubygems.org or point to master branch on Github?

**Pull request merged**